setup.py

```diff
@@ -78,22 +85,22 @@ def get_git_revision():
             stderr=subprocess.DEVNULL,
         )
         branch = result.stdout.rstrip()
-        if branch != "" and branch != "master" and not branch.endswith("/master"):
-            version = version + "-" + branch
+        if branch and branch != "main" and not branch.endswith("/main"):
+            commit = f"{commit}-{branch.rsplit('/', 1)[-1]}"
     except Exception:
         pass
 
+    version = f"{major}.{minor}.{patch}+{commit}"
     return version
 
 
-setup(
-    install_requires=[
-        "dials-data>=2.0",
-        "Jinja2",
-        "procrunner",
-        "tabulate",
-        'importlib_metadata;python_version<"3.8"',
-    ],
+setuptools.setup(
+    version=get_git_revision() or __version_tag__,
+    package_data={
+        "": ["*"],
+        "xia2": ["Data/*", "css/*", "templates/*"],
+        "xia2.Test": ["Handlers/*", "Modules/*", "regression/expected/*"],
+    },
```

Comment:
not sure this is necessary at all?
(or if it is: should likely be done differently)

Reply:
I'm attempting to lean on the side of caution to fully replicate the current contents, ignoring the questions of whether they should be here. As such, unless you have strong objections or know the "proper" way we're supposed to do this, I'd prefer to keep this part temporarily intact?

Reply:
Okay, after playing a bit, this does cover both `bdist` and `sdist`. I'm going to merge this in, but with the qualifier that if it's somewhat broken in another way, people can blame me and I will fix.